syntax = "proto3";
import "google/protobuf/descriptor.proto";
package my.proto.package;

enum PackageId
{
	UNKNOWN_MESSAGE_ID = 0;

	CONNECTION_REQUEST = 10;
	CONNECTION_RESPONSE = 11;
	CONNECTION_PING = 12;
	CONNECTION_PONG = 13;
	CONNECTION_DISCONNECT = 14;

	MOUSE_POSITION_MESSAGE = 100;
}

extend google.protobuf.MessageOptions {
	PackageId package_id = 50000;
}
